Mlem/Views/Tabs/Settings/Components/Views/About/AboutView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  About.swift
+//  Mlem
+//
+//  Created by Weston Hanners on 7/12/23.
+//
+
+import SwiftUI
+
+struct AboutView: View {
+    @EnvironmentObject var appState: AppState
+    
+    @Binding var navigationPath: NavigationPath
+    
+    var body: some View {
+        Group {
+            List {
+                Section {
+                    appHeaderView
+                        .listRowBackground(Color(.systemGroupedBackground))
+                }
+                
+                Section {
+                    Link(destination: URL(string: "https://mlem.group/")!) {
+                        Label("Website", systemImage: "globe").labelStyle(SquircleLabelStyle(color: .blue))
+                    }
+                    .buttonStyle(.plain)
+                    
+                    Button {
+                        Task {
+                            do {
+                                let request = GetCommunityRequest(account: appState.currentActiveAccount, communityId: 346)
```

Comment:
Communities have different IDs depending on which instance the user is logged into--it's 346 on sh.itjust.works, but 81081 on lemmy.ml